File: prob09/prob9.rb
def generate_random(num)
	rand_nums = []
	(0..num).each do |x|
		rand_nums << rand(1..100)
	end
	print rand_nums.sort
	puts "\n"
end


generate_random(10)
generate_random(20)
generate_random(5)

File: prob09/prob9.py
from random import randrange

rand_nums = []

# generate n random numbers
def generate_random(num):
	for x in range(0,num):
		rand_nums.append(randrange(100))
	sort_list(rand_nums)

# sort
def sort_list(lst):
	rand_nums.sort()
	print rand_nums

generate_random(10)
# clear the list
rand_nums = []
generate_random(20)
rand_nums = []
generate_random(5)

